include/aikido/constraint/CollisionFree.hpp

```diff
@@ -39,9 +40,16 @@ class CollisionFree : public Testable
   // Documentation inherited.
   statespace::StateSpacePtr getStateSpace() const override;
 
-  // Documentation inherited.
+  // Documentation inherited. As an important note, outcome is expected to be
```

Comment:
Nit: It would be great to expose this note by making this doxygen comment as:
```c++
/// \copydoc Testable::isSatisfied()
/// \note Outcome is expected to be an instance of ...
```
By doing above, we could see this note in addition to the original docstring of `Testable::isSatisfied()`.